doc/install/fast-install-and-first-run.rst

```diff
@@ -5,50 +5,78 @@ This file describes the steps to install IVRE, run the first scans and
 
 For another way to run IVRE easily (probably even more easily), see
 :ref:`install/docker:Docker`.
 
 Install
 -------
+External programs
+~~~~~~~~~~~~~~~~~
+
+- MongoDB Community Edition (latest version)
+- Nmap
+- Zeek/Bro (optionnal)
+- Argus (optionnal)
+- Nfdump (optionnal)
+
+::
+
+   $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+   $ echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
+   $ wget -O- "http://download.opensuse.org/repositories/network:/bro/xUbuntu_18.04/Release.key" | sudo apt-key add - # If you use Ubuntu 18.04 LTS.
+   $ echo 'deb http://download.opensuse.org/repositories/network:/bro/xUbuntu_18.04/ /' | sudo tee '/etc/apt/sources.list.d/bro.list' 
+   $ wget -O- "http://download.opensuse.org/repositories/network:/bro/Debian_10/Release.key" | sudo apt-key add - # If you use Debian 10.
+   $ echo 'deb http://download.opensuse.org/repositories/network:/bro/Debian_10/ /' | sudo tee '/etc/apt/sources.list.d/bro.list' 
+   $ sudo apt update
+   $ sudo apt install mongodb-org nmap bro argus-server nfdump gcc make libpcap-dev -y
+   $ sudo systemctl enable mongod
+   $ sudo systemctl start mongod
+   $ export PATH="$PATH:/opt/bro/bin"
+
+Append ``/opt/bro/bin`` to PATH variable in ``/etc/environment``.
+Append ``/opt/bro/bin`` to secure_path variable in ``/etc/sudoers`` using visudo.
+  
+- Masscan (optionnal)
+- P0f (optionnal)
+  
+::
 
+   $ git clone https://github.com/robertdavidgraham/masscan.git
+   $ cd masscan
+   $ make
+   $ sudo cp bin/masscan /usr/local/bin
+   $ cd ..
+   $ wget http://lcamtuf.coredump.cx/p0f3/releases/old/2.x/p0f-2.0.8.tgz
+   $ tar -xvf p0f-2.0.8.tgz
+   $ cd p0f
+   $ make
+   $ sudo make install
+
+Python 2 build (apt)
+~~~~~~~~~~~~~~~~~~~~
 ::
 
-   $ sudo apt-get -y install mongodb python-pymongo python-crypto \
-   >   python-future python-bottle apache2 libapache2-mod-wsgi dokuwiki
+   $ sudo apt-get -y install python-pymongo python-crypto python-future python-bottle apache2 libapache2-mod-wsgi dokuwiki
    $ git clone https://github.com/cea-sec/ivre
    $ cd ivre
    $ python setup.py build
    $ sudo python setup.py install
 
-NB: if you are running Debian stable, the dokuwiki package has been
-removed (no idea why: it exists in both oldstable and testing). Run the
-following commands (or similar) if you cannot install dokuwiki, and try
-again:
-
+Python 3 build (pip)
+~~~~~~~~~~~~~~~~~~~~
 ::
 
-   $ echo 'APT::Default-Release "stable";' | \
-   >   sudo tee /etc/apt/apt.conf.d/99defaultrelease
-   $ echo 'deb http://deb.debian.org/debian testing main' | \
-   >   sudo tee -a /etc/apt/sources.list
-   $ sudo apt-get update
-
-Setup
------
+   $ sudo apt install -y python3-pip python3-dev apache2 libapache2-mod-wsgi-py3 dokuwiki
+   $ sudo -H pip3 install bottle pycrypto future pymongo
```

Comment:
Is there a reason for not using the Debian packages python3-bottle, python3-future, python3-pymongo and python3-crypto?